tests/cypress/e2e/textFieldInitializerTest.cy.ts

```diff
@@ -0,0 +1,455 @@
+                    en: 'This is the default string value',
+                    fr: 'This is the default string value',
+                    de: 'This is the default string value'
+                }
+            },
+            'jnt:textFieldInitializer_moduleI18nRBString': {
+                type: 'smallText',
+                values: {
+                    en: 'Hello',
+                    fr: 'Bonjour',
+                    de: 'Guten Tag'
+                }
+            },
+            'jnt:textFieldInitializer_moduleRBAutocreatedString': {
+                type: 'smallText',
+                values: {
+                    en: 'Jahia rocks',
+                    fr: 'Jahia rocks',
+                    de: 'Jahia rocks'
+                }
+            },
+            'jnt:textFieldInitializer_moduleI18nRBAutocreatedString': {
+                type: 'smallText',
+                values: {
+                    en: 'Kiss',
+                    fr: 'Bisous',
+                    de: 'Kussen'
+                }
+            }
+        };
+
+        const valuesToCheck = {
+            fr: ['1988-03-07T19:40:00.000', '2008-03-07T19:40:00.000', '2013-03-02T19:00:00.000'],
+            de: ['1988-03-07T19:40:00.000', '2008-03-07T19:40:00.000', '2013-03-02T19:00:00.000'],
+            en: ['1988-03-07T19:40:00.000', '2006-03-07T19:40:00.000', '2008-03-07T19:40:00.000', '2013-03-02T19:00:00.000']
+        };
+
+        cy.login();
+        const pageComposer = PageComposer.visit(siteKey, langEN, 'home.html');
+        const contentEditor = pageComposer
+            .openCreateContent()
+            .getContentTypeSelector()
+            .searchForContentType('textFieldInitializer')
+            .selectContentType('textFieldInitializer')
+            .create();
+        checkFieldValues(contentEditor, fieldsToCheck, langEN);
+        contentEditor.getSmallTextField('mix:title_jcr:title').addNewValue('englishTitle', true);
+        contentEditor.getLanguageSwitcher().selectLang('Français');
+        checkFieldValues(contentEditor, fieldsToCheck, langFR);
+        contentEditor.getSmallTextField('mix:title_jcr:title').addNewValue('frenchTitle', true);
+        contentEditor.getLanguageSwitcher().selectLang('Deutsch');
+        checkFieldValues(contentEditor, fieldsToCheck, langDE);
+        contentEditor.getSmallTextField('mix:title_jcr:title').addNewValue('deutschTitle', true);
+        contentEditor.getLanguageSwitcher().selectLang('English');
+        contentEditor.create();
+        pageComposer.switchLanguage('Français');
+        checkValuesDisplayedInPageComposer(pageComposer, valuesToCheck, langFR);
+        pageComposer.switchLanguage('Deutsch');
+        checkValuesDisplayedInPageComposer(pageComposer, valuesToCheck, langDE);
+        pageComposer.switchLanguage('English');
+        checkValuesDisplayedInPageComposer(pageComposer, valuesToCheck, langEN);
+    });
+
+    it('Verify the edition of previous text initializer', () => {
```

Comment:
This test seems to be dependent to the previous one, it's would be better to have separate logic between each tests. 
You should create the necessary contents at the beginning of the test and delete it at the end

Reply:
Fixed